include/cinatra/coro_http_client.hpp

```diff
@@ -1680,6 +1689,7 @@ class coro_http_client {
   std::unique_ptr<asio::ssl::stream<asio::ip::tcp::socket &>> ssl_stream_;
   bool ssl_init_ret_ = true;
   bool use_ssl_ = false;
+  std::string sni_hostname_ = "";
```

Comment:
also add CINATRA_ENABLE_SSL with sni_hostname_?

Reply:
yeah, only https need this it's tls extension.